src/common/utils.tsx

```diff
@@ -26,6 +28,23 @@ interface QrCode {
   };
 }
 
+export function readFileAsCsv<T>(file: File, headers?: string[]): Promise<Array<T>> {
+  return new Promise((resolve, reject) => {
+    const reader: FileReader = new FileReader();
+    if (reader.error) {
+      reject(reader.error);
+    }
+    reader.onload = async () => {
```

Comment:
csvtojson seems to have a await/sync api now, think you should use that instead of wrapping it yourself

![image](https://user-images.githubusercontent.com/27843274/105667630-980f7080-5f16-11eb-9b92-92bbc36f8db4.png)

Reply:
Done